src/modules/mavlink/mavlink_messages.cpp

```diff
@@ -718,7 +718,7 @@ class MavlinkStreamHighresIMU : public MavlinkStream
 
 			if (_baro_timestamp != air_data.timestamp) {
 				/* mark last group dimensions as changed */
-				fields_updated |= (1 << 9) | (1 << 11) | (1 << 12);
+				fields_updated |= (1 << 9) | (1 << 10) | (1 << 11) | (1 << 12);
```

Comment:
right -- i asked about this in the initial comment. I was only uncertain exactly how you may wish implement this -- does it then need it's own timestamp corresponding to the diff pres updated? (instead of sharing with the baro timestamp and the other three members?)

Reply:
Yes, duplicate the _baro_timestamp usage for differential_pressure (see a few lines below).